src/commands/install.yml

```diff
@@ -103,8 +110,8 @@ steps:
             "$JQ_BINARY_URL"
         fi
 
-        $SUDO mv "$jqBinary" <<parameters.install-dir>>/jq
-        $SUDO chmod +x <<parameters.install-dir>>/jq
+        mv "$jqBinary" <<parameters.install-dir>>/jq
```

Comment:
The `$SUDO` variable is still needed. The default for `install-dir` was changed to `~/bin` in this PR for which this will work fine. Since it's a parameter, what happens if someone passes a PATH that does need to root/sudo access? Such as the original path `/usr/local/bin`. This line and the following will still fail.

Reply:
Updated this to check if we do need to export `$SUDO` or not :)